From efd2c8eebe4baaccfe549772de55add00e85fe04 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Tue, 26 Nov 2024 15:10:39 +0800
Subject: [PATCH 284/303] LoongArch: NFC: Dedup and sort the comment in
 loongarch_print_operand_reloc

gcc/ChangeLog:

        * config/loongarch/loongarch.cc (loongarch_print_operand_reloc):
        Dedup and sort the comment describing modifiers.
---
 gcc/config/loongarch/loongarch.cc | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/src/gcc/config/loongarch/loongarch.cc b/src/gcc/config/loongarch/loongarch.cc
index e1be6e768..bb3769f74 100644
--- a/src/gcc/config/loongarch/loongarch.cc
+++ b/src/gcc/config/loongarch/loongarch.cc
@@ -6128,21 +6128,13 @@ loongarch_print_operand_reloc (FILE *file, rtx op, bool hi64_part,
    'T'	Print 'f' for (eq:CC ...), 't' for (ne:CC ...),
 	      'z' for (eq:?I ...), 'n' for (ne:?I ...).
    't'	Like 'T', but with the EQ/NE cases reversed
-   'F' Print the FPU branch condition for comparison OP.
-   'W' Print the inverse of the FPU branch condition for comparison OP.
-   'w' Print a LSX register.
    'u' Print a LASX register.
-   'T' Print 'f' for (eq:CC ...), 't' for (ne:CC ...),
-             'z' for (eq:?I ...), 'n' for (ne:?I ...).
-   't' Like 'T', but with the EQ/NE cases reversed
-   'Y' Print loongarch_fp_conditions[INTVAL (OP)]
-   'Z' Print OP and a comma for 8CC, otherwise print nothing.
-   'z' Print $0 if OP is zero, otherwise print OP normally.  
    'v' Print the insn size suffix b, h, w or d for vector modes V16QI, V8HI,
          V4SI, V2SI, and w, d for vector modes V4SF, V2DF respectively.
    'V' Print exact log2 of CONST_INT OP element 0 of a replicated
          CONST_VECTOR in decimal.	 
    'W' Print the inverse of the FPU branch condition for comparison OP.
+   'w' Print a LSX register.
    'X' Print CONST_INT OP in hexadecimal format.
    'x' Print the low 16 bits of CONST_INT OP in hexadecimal format.
    'Y' Print loongarch_fp_conditions[INTVAL (OP)]
-- 
2.33.0

